CONTRIBUTING.md

```diff
@@ -1,36 +1,107 @@
-# How to contribute
+# Contributing Guide
 
-## Install dependencies
+Thank you for your interest in contributing to the **node-nepali-datetime** project!
+Your contributions will help improve and enhance this library.
+Please take a moment to review the following guidelines before getting started.
 
-```bash
-npm install
-```
+## Prerequisites
 
-## Codestyle
+Before contributing, ensure that you have the following:
 
-A [.prettierrc](./.prettierrc) file is available to maintain the coding style. Besides, your code will automatically be formatted by the pre-commit hook.
+-   **Node.js v18 or higher** installed. Download it from the [official Node.js website](https://nodejs.org/)
+    or use `nvm` to manage multiple Node.js versions.
+-   **npm** installed (comes with Node.js).
 
-## Unit tests
+## Getting Started
 
-Run the unittest using the below command:
+To set up the project on your local machine, follow these steps:
 
-```bash
-npm test
-```
+1. **Fork** the repository on GitHub.
+2. **Clone** the forked repository to your local machine:
 
-## Before submitting
+    ```bash
+    git clone https://github.com/<your-username>/node-nepali-datetime.git
+    cd node-nepali-datetime
+    ```
 
-Before submitting your Pull Request, please do the following steps:
+3. **Install dependencies**:
 
-1. Add any changes you want.
-1. Add tests for the new changes.
-1. Edit documentation (`README.md`) if you have changed something significant.
-1. Commit your changes using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).  
-   Examples: `feat: add JSON parser`, `feat(parser): add JSON parser`.
+    ```bash
+    npm install
+    ```
 
-## Other help
+4. **Verify your setup**:
 
-You can contribute by spreading a word about this library.
-It would also be a huge contribution to write
-a short article on how you are using this project.
-You can also share your best practices with us.
+    - Run the example script to test the library:
+
+        ```bash
+        npm run example
+        ```
+
+        Modify the [example.ts](./example.ts) file to experiment with your example code.
+
+    - Run lint checks:
+
+        ```bash
+        npm run lint
+        ```
+
+    - Run tests to ensure everything works:
+
+        ```bash
+        npm run test
+        ```
+
+5. **Start contributing** by making the necessary changes to the codebase.
+
+## Code Style
+
+-   A **Prettier** configuration file ([`.prettierrc`](./.prettierrc)) is provided to maintain a consistent coding style.
+-   Pre-commit hooks will automatically format your code before committing. Ensure that your code passes lint checks.
+
+## Pull Requests
+
+We welcome and appreciate pull requests from the community. To contribute:
+
+1. **Fork** the repository and create a new branch based on the `main` branch:
```

Comment:
Suggestion: To create a new branch, you can also use the `create a branch` feature from the issues page. Let's include that as well here.

![image](https://github.com/user-attachments/assets/57a0d5fb-e70d-4fd4-a206-7c98c5eaedfa)

Reply:
oh  ok.